part-iv-advanced-tuning/advanced-customizations.md

```diff
@@ -154,6 +153,12 @@ public class MethodCommandHandlerDefinition implements HandlerEnhancerDefinition
 
 To skip all handling of the handler then just throw an exception.
 
+Since 3.3 Axon version it is possible to configure `HandlerDefinition` with Axon `Configuration`. If you are using Spring Boot defining `HandlerDefintion`s and `HandlerEnhancerDefinition`s as beans is sufficient (Axon autoconfiguration will pick them up and configure within Axon `Configuration`).
+
+```java
+configurer.registerHandlerDefinition(c -> new MethodCommandHandlerDefinition());
```

Comment:
Might be nice to state `Configurer configurer = ...` or provide it as a parameter to an example configure method, as I think it is beneficial to be a bit more specific.